sops/__init__.py

```diff
@@ -545,6 +558,9 @@ def verify_or_create_sops_branch(tree, kms_arns=None, pgp_fps=None,
         tree, has_at_least_one_method = parse_kms_arn(tree, kms_arns)
     if pgp_fps:
         tree, has_at_least_one_method = parse_pgp_fp(tree, pgp_fps)
+    if context:
+        for entry in tree['sops']['kms']:
+            entry['context'] = context
```

Comment:
I believe this is meant to be a dict. I'd suggest validating its format before storing it.